soh/src/overlays/actors/ovl_En_Kz/z_en_kz.c

```diff
@@ -125,43 +125,42 @@ u16 EnKz_GetText(PlayState* play, Actor* thisx) {
 
 s16 func_80A9C6C0(PlayState* play, Actor* thisx) {
     EnKz* this = (EnKz*)thisx;
-    s16 ret = NPC_TALK_STATE_TALKING;
+    s16 talkState = NPC_TALK_STATE_TALKING;
 
     switch (Message_GetState(&play->msgCtx)) {
         case TEXT_STATE_DONE:
             if (CVarGetInteger(CVAR_ENHANCEMENT("EarlyEyeballFrog"), 0)) {
                 if (Message_ShouldAdvance(play)) {
-                    ret = NPC_TALK_STATE_ITEM_GIVEN;
+                    talkState = NPC_TALK_STATE_ITEM_GIVEN;
                 }
             } else {
-                ret = NPC_TALK_STATE_IDLE;
+                talkState = NPC_TALK_STATE_IDLE;
                 switch (this->actor.textId) {
                     case 0x4012:
-                        Flags_SetInfTable(INFTABLE_139);
-                        ret = NPC_TALK_STATE_ACTION;
+                        SET_INFTABLE(INFTABLE_139);
```

Comment:
These `SET_INFTABLE` need to remain as `Flags_SetInfTable` (cause it contains our hook callbacks).
Unlike 2ship, we didn't replace the macro definitions to call our functions, instead we replaced all the source code instances to use the func directly.